InstagramClone/scripts/main_v2.js

```diff
@@ -0,0 +1,84 @@
+var imgArray = [
+    {url:'https://yt3.ggpht.com/a-/AJLlDp2Z1S4_t4FFXDJ8GIY8a4u1QrxcB0j3ruoH9w=s900-mo-c-c0xffffffff-rj-k-no',
+    caption: 'Look at this image',
+    comments: ['That is neat!','Love Golum!']},
+    {url:'https://c1.staticflickr.com/8/7270/7554426478_da09815d9e_b.jpg',
+    caption: 'Look at this image',
+    comments: ['That is neat!','Love Golum!']},
+    {url:'http://www.freakingnews.com/pictures/105000/Kate-Bosworth-as-Golum-105467.jpg',
+    caption: 'Look at this image',
+    comments: ['That is neat!','Love Golum!']},
+    {url:'http://newsrescue.com/wp-content/uploads/2017/04/gollum.jpg',
+    caption: 'Look at this image',
+    comments: ['That is neat!','Love Golum!']}
+]
+//loops through array
+imgArray.forEach(function(image) { 
+    //creates and adds img for all imgs in Array
+    (function (){
+        var imageTag = document.createElement('img')
+        imageTag.src = image.url
+        var container = document.querySelector(".container"); 
+        container.appendChild(imageTag);
+
+        imageTag.addEventListener('click',function(event){
+            var imageSelect = document.querySelector('img');
+            var className = imageSelect.getAttribute("class");
+            if(className=="lightbox") {
+                imageSelect.className = "lightbox.on";
```

Comment:
I think class names don't have a '.'  If you want to make two classnames on this element, syntax is className = "lightbox on"